pyproject.toml

```diff
@@ -11,6 +11,7 @@ requires = [
     "typing_extensions",
     "six",
     "requests",
+    "optree>=0.5.1",
```

Comment:
Is this actually needed as a build-requirement? I don't think so, it can be installed as a project requirement.

Reply:
Yeah it should be installed in setup.py, but it doesn't need to be in the isolated build env